test_user.py

```diff
@@ -8,23 +8,23 @@ def test_when_player_is_created_it_has_a_symbol():
 def test_valid_choice_is_returned():
     game_board = Board()
     game_board.new_game()
-    assert spot_validation(game_board.board_state, "1") == 1
+    assert UserActions.make_move(game_board.board_state, "1") == 1
     
 def test_invalid_symbol_returns_false():
     game_board = Board()
     game_board.new_game()
-    assert spot_validation(game_board.board_state, "x") == False 
+    assert UserActions.make_move(game_board.board_state, "x") == False 
```

Comment:
So this function returns both an `int` and a `boolean`? interesting...

Reply:
It returns both because it's being used to check whether a spot is valid, and convert the user input to a int. The function returns a boolean if the users selection was invalid, other wise it returns the correctly formatted input.

I created this because all user input when dealing with the command line comes in as a string (even numerical selections). In hindsight, the conversion should be split off so that the function isn't returning two types.